pkg/controllers/dynakube/injection/conditions.go

```diff
@@ -32,3 +46,82 @@ func setCodeModulesInjectionCreatedCondition(conditions *[]metav1.Condition) {
 	}
 	_ = meta.SetStatusCondition(conditions, condition)
 }
+
+func setNamespacesMonitoredSelectorCondition(conditions *[]metav1.Condition, selectorType string, configured bool, names []string) {
+	var condType string
+
+	switch selectorType {
+	case "OneAgent":
+		condType = oneAgentNamespacesMonitoredConditionType
+	case "MetadataEnrichment":
+		condType = metadataEnrichmentNamespacesMonitoredConditionType
+	default:
+		condType = namespacesMonitoredConditionType
```

Comment:
what is the purpose of this?

currently it will always be 0, because `default` only runs when non of the cases match

Reply:
I thought about it being a general info to inform if namespaces match configured selectors, what do you mean by it always being 0? 

EDIT: I see it now I will remove it 😅